aten/src/ATen/native/special_functions/detail/sinhc.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace at {
+namespace native {
+namespace special_functions {
+namespace detail {
```

Comment:
I wish we can use C++17 nested namespaces, but for the time being, wouldn't it be more compact to define them all on one line (which is a common convention for many Aten/native headers)
```suggestion
namespace at { namespace native { namespace special_functions { namespace detail {
```